dc

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ -f .env ]; then
+  export $(cat .env | sed 's/#.*//g' | xargs)
+fi
+
+for i in $(seq 1 ${#PORTAL_MODULES}); do
+  # accounts module - alias "a"
+  if [ ${PORTAL_MODULES:i-1:1} == "a" ]; then
+    COMPOSE_FILES+=" -f docker-compose.accounts.yml"
+  fi
+
+  # jaeger module - alias "j"
+  if [ ${PORTAL_MODULES:i-1:1} == "j" ]; then
+    COMPOSE_FILES+=" -f docker-compose.jaeger.yml"
+  fi
+done
```

Comment:
Couldn't be simplified to something like?

`echo "${#PORTAL_MODULES}" | grep a && COMPOSE_FILES+=" -f docker-compose.accounts.yml"`
`echo "${#PORTAL_MODULES}" | grep j && COMPOSE_FILES+=" -f docker-compose.jaeger.yml"`

Reply:
I wanted to make sure we include modules in an order that they are defined in the PORTAL_MODULES that's why I need a loop (sometimes one file would override some settings from the other and order is important)

Reply:
we should align this in ansible